action-scheduler.php

```diff
@@ -25,28 +25,28 @@
  *
  */
 
-if ( ! function_exists( 'action_scheduler_register_3_dot_1_dot_6' ) ) {
+if ( ! function_exists( 'action_scheduler_register_3_dot_2_dot_0' ) ) {
 
 	if ( ! class_exists( 'ActionScheduler_Versions' ) ) {
 		require_once( 'classes/ActionScheduler_Versions.php' );
 		add_action( 'plugins_loaded', array( 'ActionScheduler_Versions', 'initialize_latest_version' ), 1, 0 );
 	}
 
-	add_action( 'plugins_loaded', 'action_scheduler_register_3_dot_1_dot_6', 0, 0 );
+	add_action( 'plugins_loaded', 'action_scheduler_register_3_dot_2_dot_0', 0, 0 );
 
-	function action_scheduler_register_3_dot_1_dot_6() {
+	function action_scheduler_register_3_dot_2_dot_0() {
 		$versions = ActionScheduler_Versions::instance();
-		$versions->register( '3.1.6', 'action_scheduler_initialize_3_dot_1_dot_6' );
+		$versions->register( '3.1.6', 'action_scheduler_initialize_3_dot_2_dot_0' );
```

Comment:
`$versions->register( '3.1.6'` 
should be 
`$versions->register( '3.2.0'`

The version that is loaded is determined by sorting registered version descending. The first callback for a registered version is the one that is used to load it if it is the highest verson.

Reply:
Nice catch, added in https://github.com/woocommerce/action-scheduler/pull/703/commits/2a43795a99bebff2cd5fcc82ad26d3bc0330b8f1.